### ISSUE TITLE
VMProbe module incorrect configuration

### ISSUE THREAD

**Issue by GagarinX** [VictoriaMetrics/operator#125]
Hello.

When I apply this VMProbe:

```
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMProbe
metadata:
  name: example.com
spec:
  jobName: web_check_hc
  vmProberSpec:
    url: blackbox-exporter.blackbox-exporter.svc.cluster.local:9115
    scheme: http
    path: /probe
  module: http_hc
  targets:
    staticConfig:
      targets:
        - "https://example.com/"
```

I get this job in VMAgent config (vmagent.env.yaml):

```
- job_name: vm-mon/example.com/0
  scheme: http
  metrics_path: /probe
  module: http_hc
  static_configs:
  - targets:
    - https://example.com/
  relabel_configs:
  - target_label: job
    replacement: web_check_hc
  - source_labels:
    - __address__
    target_label: __param_target
  - source_labels:
    - __param_target
    target_label: instance
  - target_label: __address__
    replacement: blackbox-exporter.blackbox-exporter.svc.cluster.local:9115
```

Which doesn't work because module should be configured in "params" like this:

```
    params:
      module: [http_hc] 
```

Instead there is just a line `module: http_hc`.

**Comment by f41gh7** [comment]
Thanks for report, must be fixed at master branch.

**Comment by f41gh7** [comment]
fix added to the latest v0.6.0 release!
